src/Particle/ParticleTile.cpp

```diff
@@ -66,17 +76,19 @@ void make_ParticleTile(py::module &m, std::string allocstr)
         .def("GetStructOfArrays", py::overload_cast<>(&ParticleTileType::GetStructOfArrays),
             py::return_value_policy::reference_internal)
         .def("empty", &ParticleTileType::empty)
-        .def("size", &ParticleTileType::size)
-        .def("numParticles", &ParticleTileType::numParticles)
-        .def("numRealParticles", &ParticleTileType::numRealParticles)
-        .def("numNeighborParticles", &ParticleTileType::numNeighborParticles)
-        .def("numTotalParticles", &ParticleTileType::numTotalParticles)
+        //.def("size", &ParticleTileType::size) // TODO
+        //.def("numParticles", &ParticleTileType::numParticles) // TODO
+        //.def("numRealParticles", &ParticleTileType::numRealParticles) // TODO
+        //.def("numNeighborParticles", &ParticleTileType::numNeighborParticles) // TODO
+        //.def("numTotalParticles", &ParticleTileType::numTotalParticles) // TODO
```

Comment:
To check: Did we accidentally drop these or are they now at a different location (on `ParticleType`?)

Reply:
Oh, I think they belong on the particle tile, not the particle tile data...